test/extended/core.sh

```diff
@@ -81,6 +80,8 @@ DEFAULT_SKIP=$(join '|' "${SKIP_TESTS[@]}")
 SKIP="${SKIP:-$DEFAULT_SKIP}"
 
 if [[ -z ${TEST_ONLY+x} ]]; then
```

Comment:
what is this TEST_ONLY flag?  this is a test script...  and why would iptables not always be required for these tests given that they launch a full openshift server?